Traceback (most recent call last):
  File "C:\Users\Gold System\AppData\Local\Temp\_MEI115962\cenariostestespegasus\CadastrosPrincipais\CadastrosCenáriosCestaBásica\cadastrodecestabasica3ºcenario.py", line 325, in <module>
    main()
    ~~~~^^
  File "C:\Users\Gold System\AppData\Local\Temp\_MEI115962\cenariostestespegasus\CadastrosPrincipais\CadastrosCenáriosCestaBásica\cadastrodecestabasica3ºcenario.py", line 275, in main
    if not safe_action(doc, "Acessando o sistema", lambda: driver.get(URL), driver, wait)[0]:
           ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: safe_action() takes 3 positional arguments but 5 were given
